### ISSUE TITLE
[Bug report] `JacocoReport` does not work  

### ISSUE THREAD

**Issue by yuqi1129** [datastrato/gravitino#1488]
### Describe what's wrong

The test coverage tool `JacocoReport` did not work at some point, we need to make it run normally. 

### Error message and/or stacktrace

<img width="1391" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/2a9f255e-e498-4f34-8e98-9228dde375f8">

### How to reproduce

Please see above.

**Comment by hiirrxnn** [comment]
I would like to work on this . Please tell me precisely what is to be done here.

**Comment by justinmclean** [comment]
Unless you have experience with the tool and GitHub plugin it might be best to select another issue to work on.

**Comment by yuqi1129** [comment]
@hiirrxnn 
Hi, are you still interested in this issue?

**Comment by yijhenlin** [comment]
It seems like the file `integration-test-common/build/reports/jacoco/test/jacocoTestReport.xml` is incorrect. (There was a similar issue in Madrapps/jacoco-report)
I'm not sure if we need the report of integration-test-common, since I saw the workflow use the arg `-PskipITs`.
If not, maybe this issue can be resolved by ignoring integration-test-common when generating reports. (or skipITs should skip this module?)
![Snipaste_2024-03-29_20-43-08](https://github.com/datastrato/gravitino/assets/44198696/177f66f9-4c04-438b-a100-0bdf273fc082)

**Comment by justinmclean** [comment]
@yuqi1129 Are you still working on this?

**Comment by yuqi1129** [comment]
> [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/15794564?u=0045ee306d34759b726347d36a743e8c7c81f097&amp;v=4&amp;size=48">@yuqi1129](https://github.com/yuqi1129) Are you still working on this?

Yeah, I'm communicating with the ASF community and seeking their help.

**Comment by mchades** [comment]
Fixed by #10087
